### DISCUSSION TITLE
Apply style to text selection in RTE instead of individual lines

### DISCUSSION THREAD

**Discussion by RasmusUmbraco** [umbraco/Umbraco-CMS#14443] [Features and ideas]
When styling text in the RTE the style gets applied to each line of text.
This can mess quite a bit with the style and make it look nothing like the preview.
Would it be possible to apply the style to the selection instead of applying it to each line? 

In the preview:
![image](https://github.com/umbraco/Umbraco-CMS/assets/88326257/1dd5ce76-a6df-4472-b833-d68b3771b3de)

In the RTE:
![image](https://github.com/umbraco/Umbraco-CMS/assets/88326257/51f23b32-c31b-4982-ad22-6ed346a0922d)

In the code:
![image](https://github.com/umbraco/Umbraco-CMS/assets/88326257/8b303f85-98af-4f92-ab9c-7c8556256756)

**Comment by iOvergaard** [comment]
We don't really control how TinyMCE handles selection. Still, I would try to make sure that your lines are in fact only present in one set of `<p>...</p>` tags before you apply the styles because Tiny will reapply any styles to anything it deems as a new "block", which a `<p>` tag certainly would fall under.

**Reply by iOvergaard** [reply]
For instance, create new lines using `SHIFT` + `ENTER` which should leave the text as one "block" and then apply the styes. See if that helps?
